content/doc/developer/javadoc.html.haml

```diff
@@ -27,7 +27,7 @@ title: Javadoc
 %ul
   - previous = ''
   -# No more than 15 LTS releases because update center does not generally extend beyond 12 months
```

Comment:
Could we update the comment to match the code?
```suggestion
  -# No more than 9 LTS releases because LTS baseline recommends most recent baseline and two baselines prior
```